src/styles/index.scss

```diff
@@ -2,2 +1,231 @@
+.page {
   margin: 0;
+  box-sizing: border-box;
+  scroll-behavior: smooth;
+}
+
+:root {
+  --main-text-color: #00acdc;
+}
+
+.header {
+  margin: 0;
+  text-transform: uppercase;
+  box-sizing: border-box;
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding-inline: 50px;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+  display: flex;
+}
+
+
+.nav__list {
+  justify-content: center;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+}
+
+.nav__link {
+  color: #060b35;
+  align-items: center;
+  height: 60px;
+  margin-left: 20px;
+  text-decoration: none;
```

Comment:
The `align-items: center;` style is not needed here. The `align-items` property defines how the browser distributes space between and around content items along the cross-axis of a flex container. This is more relevant when you have multiple lines of child elements inside a flex container. In this case, you have only one line, so this property has no effect.